### ISSUE TITLE
[PXP - birth date confirmation] The group of form controls is not associated with its group label.

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5831]
## Area or Component

PXP flow -> Birth date confirmation page

## Quick Summary

Birth date confirmation contains a three text fields that in conjunction represent the birth date information. Currently there is a global label `Date of birth` that is not associated to the three text fields below. 

Other issues identified are:

- The form is not wrapped in a `<form>` tag.
- Focus is not moved to field with error.

### How to Replicate

Go to /pxp/birth-date-confirmation?plid=[plid_number] and check the DOM. 

## Accessibility Violation

[The group of form controls is not associated with its group label. The group-level label helps to completely convey the purpose of individual controls in the group.](https://docs.deque.com/issue-help/1.0.0/en/label-group-not-associated)

## Action Requested

Implement one the patterns described in [this dequeue training](https://dequeuniversity.com/class/forms-wcag-2.2/labels/one-label-multiple-fields). Based on the current structure of the form the fieldset + legend looks the best fit.

Add a `<form>` tag to the form and implement the focus on error.

## Screenshots (optional)
![Screenshot 2023-05-17 at 11 48 30 AM](https://github.com/CDCgov/prime-simplereport/assets/103958711/26981238-7905-4e2d-acaa-a5b87e867fa2)
